### PR TITLE
Add options for all glitched logic tricks

### PR DESCRIPTION
**Pull request by Pronyo-Chan**

### CODE REVIEW

**Review by Pronyo-Chan — COMMENTED** [review]
lgtm

File: maps/graph_edges/glitched_logic/nok_bombetteless_kbf_fp_plus.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Bombette-less KBF FP Plus.
+"""
+edges_nok_add_bombetteless_fp_plus_lzs = [
+    #? Path to Fortress 2 Exit Bottom Right -> Path to Fortress 2 Top Pipe
```
Comment (icebound777) [inline maps/graph_edges/glitched_logic/nok_bombetteless_kbf_fp_plus.py:6]: id 2 is -> "Path to Fortress 2 Exit Top Right"

File: maps/graph_edges/glitched_logic/nok_bombetteless_kbf_fp_plus.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {"from": {"map": "NOK_15", "id": 1}, "to": {"map": "NOK_15", "id": 2}, "reqs": [], "mapchange": False},
+]
+
+edges_nok_add_bombetteless_fp_plus_laki = [
+    #? Path to Fortress 2 Exit Bottom Right -> Path to Fortress 2 Top Pipe
```
Comment (icebound777) [inline maps/graph_edges/glitched_logic/nok_bombetteless_kbf_fp_plus.py:11]: id 2 is -> "Path to Fortress 2 Exit Top Right"

File: maps/graph_edges/glitched_logic/obk_parakaryless_boos_portrait.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+edges_obk_add_boo_portrait_laki = [
+    #* Library Bombable Wall -> ItemA (BooPortrait)
+    {"from": {"map": "OBK_06", "id": 1}, "to": {"map": "OBK_06", "id": "ItemA"}, "reqs": [["Lakilester"]], "mapchange": False},
+    #* ItemA (BooPortrait) -> Library Fall From Ceiling
+    {"from": {"map": "OBK_06", "id": "ItemA"},  "to": {"map": "OBK_06", "id": 0}, "reqs": [["Lakilester"]], "mapchange": False},
+    #* Library Bombable Wall -> Library Fall From Ceiling
+    {"from": {"map": "OBK_06", "id": 1}, "to": {"map": "OBK_06", "id": 0}, "reqs": [["Lakilester"]], "mapchange": False},
+    #? Library Fall From Ceiling -> Library Bombable Wall
+    {"from": {"map": "OBK_06", "id": 0}, "to": {"map": "OBK_06", "id": 1}, "reqs": [["Lakilester"]]},
```
Comment (icebound777) [inline maps/graph_edges/glitched_logic/obk_parakaryless_boos_portrait.py:20]: add "mapchange": False

**Review by icebound777 — COMMENTED** [review]
IMO can be merged to dev once the three points are adressed

**Pull request merged**